### DISCUSSION TITLE
Multiselect with Keyboard

### DISCUSSION THREAD

**Discussion by El-Barto89** [hluk/CopyQ#2850] [Q&A] [answered]
Hello everyone, could anyone explain to me how to make a multiselection of the items present in the clipboard with the keyboard? Pressing control+space promotes the "search" function, which shifts the focus to the searchbar causing the selected items to be lost. Any suggestions?

**Comment by hluk** [comment]
Shift+Up/Down arrow selects multiple consecutive items in the item list in the main window.

**Comment by El-Barto89** [comment]
Yes thank you, but what im trying to do isn't a consecutive selection. Im trying to select items like you do into a file manager with `control + space `. 
Is it any chance?

**Reply by hluk** [reply]
Oh, interesting, I never had to use it in CopyQ. So it seems that Ctrl+Space should toggle selecting the current items, but currently it also opens the search (which is incorrect). Let me try to fix that. And thanks for the report! :+1:

**Reply by El-Barto89** [reply]
yeah it's my pleasure! thank you for your amazing tool.

**Reply by hluk** [reply, marked as answer]
Looks like it is probably a Qt bug. Fix for CopyQ: #2867

**Reply by El-Barto89** [reply]
My bad for not thanking you earlier! Being new to Linux, I didn't realize you'd already solved everything and I just needed to build from source. It works like a charm - thanks again!"
